.github/workflows/virtual-cluster.yml

```diff
@@ -0,0 +1,33 @@
+name: sriov-operator-test
+on: [pull_request]
+
+jobs:
+  virtual-k8s-cluster:
+    name: k8s
+    runs-on: [sriov]
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
+
+      - name: Set up Go 1.20
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.20.x
+
+      - name: run test
+        run: make test-e2e-conformance-virtual-k8s-cluster-ci
+
+  virtual-ocp:
+    name: ocp
+    runs-on: [ ocp ]
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
+
+      - name: Set up Go 1.20
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.20.x
+
+      - name: run test
+        run: make test-e2e-conformance-virtual-ocp-cluster-ci
```

Comment:
Can you add a step to upload the reports?

something like
```
    - name: Upload report
      if: always()
      uses: actions/upload-artifact@v3
      with:
        name: k8s-reporter
        path: /tmp/artifacts
```

Reply:
yep I was working on that :)